DataFormats/SoATemplate/interface/SoAView.h

```diff
@@ -554,7 +554,7 @@ namespace cms::soa {
       Metadata(const Metadata&) = delete;                                                                              \
                                                                                                                        \
     private:                                                                                                           \
-      SOA_HOST_DEVICE SOA_INLINE Metadata(const VIEW& parent) : parent_(parent) {}                                     \
+      SOA_HOST_DEVICE SOA_INLINE Metadata(const VIEW& _soa_impl_parent) : parent_(_soa_impl_parent) {}                 \
```

Comment:
are you sure we need to protect _all_ these names ?
for example, I don't think this one can clash ?

Reply:
Indeed, this one was over the top (removed).